docs/src/hugo/content/versions.md

```diff
@@ -49,16 +49,16 @@ title: Versions
 	  </tr>
 	</thead>
 	<tbody>
-	  <tr>
-	    <td>0.16.0-cats-SNAPSHOT</td>
-	    <td class="text-center"><span class="label label-warning">Incomplete</span></td>
-	    <td class="text-center"><i class="fa fa-ban"></i></td>
-	    <td class="text-center"><i class="fa fa-check"></i></td>
-	    <td class="text-center"><i class="fa fa-check"></i></td>
-	    <td>cats-0.9</td>
-	    <td>fs2-0.9</td>
-	    <td>1.8+</td>
-	  </tr>
+      <tr>
+        <td>0.16.0-cats-SNAPSHOT</td>
+        <td class="text-center"><span class="label label-warning">Development</span></td>
```

Comment:
I think the swtich to "Development" was intentional, but noting it in case it was not.